data/NFTE/data.json

```diff
@@ -1,16 +1,19 @@
 {
-    "name": "NFTEarth",
+    "name": "NFTEarthOFT",
     "symbol": "NFTE",
     "decimals": 18,
-    "description": "NFTEarth is a marketplace for NFTs.",
+    "description": "NFTEarth is a decentralized NFT marketplace on built on Optimism and other networks designed for scalability.",
```

Comment:
```suggestion
    "description": "NFTEarth is a decentralized NFT marketplace built on Optimism and other networks designed for scalability.",
```

Reply:
Could you also add "bridged by Layer Zero" or something like that here to clarify this is not bridgable with the standard bridge?

Reply:
Yes! Will get this updated ASAP as you have advised.